### DISCUSSION TITLE
New contribution proposal: UI Fix - Admin screen desktop issues

### DISCUSSION THREAD

**Discussion by DevCanaan** [Shopify/polaris-react#12425] [Contribution proposals]
### TL;DR

In Shopify, there is an issue with some of the backend/admin screen loading off to the right. After going into google console, it looks like the main issue is a class called "Polaris-Frame__Scrollable" that has it's width set to "100vw" instead of "100%". 
The issue mainly causes certain buttons, graphs, etc. to run off screen on the righthand side of the display on desktop.

### Champion

Canaan

### Stakeholders

Whoever handles Web Development for the backend of Shopify.

### Estimated timeline

I don't have access to work on this, but it is a simple fix.

### What problem are you solving?

Whenever on the backend/admin side of Shopify, if you are in certain tabs (Products, Analytics, etc.) some are experiencing issues with the display running off the righthand side of the screen. It doesn't properly show the buttons, graphs/products run off screen, even sometimes happens on the "Themes" page.

### How does your solution improve Polaris?

It removes the bug that many are running into. Google Console shows the file as "https://cdn.shopify.com/shopifycloud/web/assets/v1/vite/client/en/assets/main-DzCgW_RgH2X3.css" and the class "Polaris-Frame__Scrollable" within this file has its width set to "100vw" which is causing the runoff. If it was simply changed to "100%" it would be fine and work properly.

### Have you tested and verified your solution?

Yes, it fixed it on each screen.

### What is the scope of work?

Just changing "100vw" to "100%". A simple fix that takes less than a minute.

### What is the impact on existing implementations, and do you have a plan to address it?

It fixes a bug that has been going on within Shopify.

### Are there any known blockers or risks?

No
